supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -2,6 +2,14 @@
 [[general-formatting]]
 = General formatting
 
+[[date-formats]]
+== Date formats
+
+Follow _The IBM Style Guide_  guidance of day Month year for date formats, for example 3 October 2019.
```

Comment:
I think that the two instances of "day Month year" and the one of "Month day, year" need to have some sort of styling around them. Something like quotes or italics.